### ISSUE TITLE
The window to which MicaForEveryone is applied is not displayed properly

### ISSUE THREAD

**Issue by MyAlfisti** [minusium/MicaForEveryone#43]
<img width="680" alt="failer" src="https://user-images.githubusercontent.com/2064997/154603144-59448f0f-0c92-45b3-9adb-d0d2650976a4.png">

As the title says, the display of the window to which MicaForEveryone is applied is disturbed.
Quitting MicaForEveryone does not disable it, and restarting Windows does not change the display.
Please tell me how to undo this problem.

**Comment by dongle-the-gadget** [comment]
> <img alt="failer" width="680" src="https://user-images.githubusercontent.com/2064997/154603144-59448f0f-0c92-45b3-9adb-d0d2650976a4.png">
> 
> As the title says, the display of the window to which MicaForEveryone is applied is disturbed. Quitting MicaForEveryone does not disable it, and restarting Windows does not change the display. Please tell me how to undo this problem.

Did you enable blur behind?

**Comment by MyAlfisti** [comment]
No, I have not enabled it.
Because the problem occurred with v0.0.4.1 of Latest.
And v0.1.0.3 Beta doesn't start properly and I can't side-load packages with an error message.
I was able to start v0.1.0.1 Beta and I tried to change some settings, but it didn't improve the problem.
By the way, the version of Windows 11 is 22557.1.

**Comment by dongle-the-gadget** [comment]
> No, I have not enabled it. Because the problem occurred with v0.0.4.1 of Latest. And v0.1.0.3 Beta doesn't start properly and I can't side-load packages with an error message. I was able to start v0.1.0.1 Beta and I tried to change some settings, but it didn't improve the problem. By the way, the version of Windows 11 is 22557.1.

Try this workaround, which just tells MFE to ignore Task Manager:
```
Process: "taskmgr" {
  TitleBarColor = Default
  BackdropPreference = Default
  ExtendFrameToClientArea = False
```

**Comment by MyAlfisti** [comment]
> この回避策を試してください。これは、MFEにタスクマネージャーを無視するように指示するだけです。
> 
> ```
> Process: "taskmgr" {
>   TitleBarColor = Default
>   BackdropPreference = Default
>   ExtendFrameToClientArea = False
> }
> ```

I set v0.1.0.1 Beta to ignore the task manager, but it didn't improve the problem.
Maybe this problem is not related to MicaForEveryone.
However, I'm also confused because I've never used any other application that is UI related.

**Comment by dongle-the-gadget** [comment]
Version 0.0.4.1 worked totally fine on my 22557 installed.

**Comment by MyAlfisti** [comment]
I found out the cause of the glitch.
It was caused by an application that I use to set system font changes in bulk.

The application that caused the problem.
https://github.com/Tatsu-syo/noMeiryoUI

The problem occurs with 100% reproducibility when I change the fonts with this application.
Right now, I installed v0.1.0.4 RC1 on Win11 22563.1 environment with sideloading and it is up and running.
I apologize for the inconvenience.
I'm sure this is a bug limited to the 2-byte character area where I live, but I hope it helps with development.
